auth/client/src/main/java/org/wildfly/security/auth/client/_private/ElytronMessages.java

```diff
@@ -129,6 +129,10 @@ ConfigXMLParseException xmlDuplicateAuthenticationConfigurationName(String name,
     ConfigXMLParseException xmlInvalidKeyStoreEntryType(@Param Location location, String alias, Class<?> expectedClass,
             Class<?> actualClass);
 
+    @LogMessage(level = WARN)
```

Comment:
In our Elytron team call yesterday, we discussed that it might be better to use INFO here instead of WARN.